plugins/checklist/assets/javascripts/discourse/initializers/checklist.js

```diff
@@ -124,13 +124,19 @@ export function checklistSyntax(elem, postDecorator) {
         // make the first run go to index = 0
         let nth = -1;
         let found = false;
+
         const newRaw = post.raw.replace(
-          /\[(\s|\_|\-|\x|\\?\*)?\]/gi,
+          /\[( |x)?\]/gi,
```

Comment:
Technically not part of the fix, but we only allow a whitespace and either "x" or "X".